server.js

```diff
@@ -21,6 +21,80 @@ const environment = process.env.NODE_ENV || 'development';
 const configuration = require('./knexfile')[environment];
 const database = require('knex')(configuration);
 
+const cleanTeam = (request, response, next) => {
+  const checkEmptyString = (stringProperties) => {
```

Comment:
i don't like that this is duplicated in cleanPlayer, but i got an error when i pulled the function out.
it was that can't do something after headers have been sent error

Reply:
looks like i'm still getting this error regardless.
the error reads: `Unhandled rejection Error: Can't set headers after they are sent.`
![this stackoverflow post](https://stackoverflow.com/questions/7042340/error-cant-set-headers-after-they-are-sent-to-the-client) describes the cause of the error pretty well, but i can't see a place where i'm starting a new send or already in the body or finished state.